### ISSUE TITLE
XAdESCC fails to validate a signature with XML CommitmentTypeQualifier

### ISSUE THREAD

**Issue by bsanchezb** [lovele0107/signatures-conformance-checker#35]
Dear @jccruellas ,

According to the specification, _CommitmentTypeQualifier_ element is a type of _AnyType_, therefore it may contain text information or an embedded XML:
```
<xsd:complexType name="CommitmentTypeIndicationType">
    <xsd:sequence>
        <xsd:element name="CommitmentTypeId" type="ObjectIdentifierType"/>
        <xsd:choice>
            <xsd:element name="ObjectReference" type="xsd:anyURI" maxOccurs="unbounded"/>
            <xsd:element name="AllSignedDataObjects"/>
        </xsd:choice>
        <xsd:element name="CommitmentTypeQualifiers" type="CommitmentTypeQualifiersListType" minOccurs="0"/>
    </xsd:sequence>
</xsd:complexType>
<xsd:complexType name="CommitmentTypeQualifiersListType">
    <xsd:sequence>
        <xsd:element name="CommitmentTypeQualifier" type="AnyType" minOccurs="0" maxOccurs="unbounded"/>
    </xsd:sequence>
</xsd:complexType>
```

However, while XAdESCC is able to verify a signature containing a text type _CommitmentTypeQualifier_, it fails to verify a signature with XML content enveloped into _CommitmentTypeQualifier_ element.

The returned error is NOT_FOUND_ERR.

Could you please take a look on the issue? The sample document is in the attachment.

Best regards,
Aleksandr.

[sample.zip](https://github.com/lovele0107/signatures-conformance-checker/files/9118633/sample.zip)

**Comment by bsanchezb** [comment]
Hm, maybe the problem is not related to all custom 'CommitmentTypeQualifier's, but it is the content of this particular CommitmentTypeQualifier that causes the issue. But still the problem persist.

**Comment by jccruellas** [comment]
Good afternoon,
I will take a look asap and will come to you.
Regards
Juan Carlos

**Comment by jccruellas** [comment]
Good afternoon,
I think that I have found the reason for the failure and fixed It.
Actually it was a failure in the process of registering the nodes with attribute 'Id' or 'ID' or 'id' which are defined in a non-XML namespace. In particular, it was the element <czip:initInstantPayment czip:id="signedData">.
XAdESCC did not properly manage the non-xml namespaces for these elements. I have fixed it in my local XAdESCC local copy. 
I attach a zip that contains the framework of html pages reporting the result of checking the signature that you have posted.

I will deploy this version of XAdESCC hopefully this afternoon.
Regards and thanks for reporting this issue
[sample-signed-xades-baseline-b.zip](https://github.com/lovele0107/signatures-conformance-checker/files/9133489/sample-signed-xades-baseline-b.zip)

Juan Carlos.
